build.gradle.kts

```diff
@@ -51,7 +51,7 @@ val operatorSdkVersion = properties["operatorSdkVersion"]
 val os = detectOs()
 val arch = detectHostArch()
 val dockerHubRepository = System.getenv()["DOCKER_HUB_REPOSITORY"] ?: "xebialabsunsupported"
-val releasedVersion = System.getenv()["RELEASE_EXPLICIT"] ?: "23.3.0-${
+val releasedVersion = System.getenv()["RELEASE_EXPLICIT"] ?: "24.1.0-${
```

Comment:
24.1.0, can we have properties[projectVersion], so its easier to update in gradle.properties alone.

Reply:
Ignore it. We're additionally setting releasedVersion also here.